guides/common/modules/proc_destroying-virtual-machines-on-host-delete.adoc

```diff
@@ -0,0 +1,34 @@
+[id="destroying-virtual-machines-on-host-delete"]
+= Destroying virtual machines on host delete
```

Comment:
```suggestion
[id="destroying-a-virtual-machine-when-deleting-a-host"]
= Destroying a virtual machine when deleting a host
```
Or can there be multiple VMs associated to one host?

Reply:
There can be hundreds of VMs associated with a host. 

However, I think the title could be clearer. We want the user to know that associated VMs are not automatically deleted on the external compute resource when the host entry is deleted and that this can cause problems in the future. The current title suggests that this is a procedure for "destroying" VMs but it's actually a procedure to configure the host so that its associated VMs are deleted at the same time. 

Maybe this would work - "Deleting virtual machines by deleting a host". 

Although the option is 'destroy-vm...', I would use 'delete' unless the rest of the documentation uses the term 'destroy'.

Reply:
I don't think that there can be multiple VMs associated with a single Foreman host. A host in Foreman is a single instance of "hardware"/OS.
I think it's useful to distinguish between delete (soft removal) and destroy (hard removal), even if Project uses "destroy" only in permission names (I checked).